hud-data/goog-hud.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "googHUD",
+  "author": "grey",
+  "contributors": [
```

Comment:
Please remove the contributors information, since you already have your name as the author

Reply:
You should keep author and remove contributors.